src/main/g8/src/main/scala/$package__packaged$/Server.scala

```diff
@@ -0,0 +1,40 @@
+package $package$
+
+
+import cats.effect.{ConcurrentEffect, Effect, ExitCode, IO, IOApp, Timer, ContextShift}
+import cats.implicits._
+import org.http4s.server.blaze.BlazeServerBuilder
+import org.http4s.client.blaze.BlazeClientBuilder
+import org.http4s.implicits._
+import fs2.Stream
+import scala.concurrent.ExecutionContext.global
+
+import org.http4s.server.middleware.Logger
+
+object Server {
```

Comment:
I think I might name this `${app.name}` instead of `Server`.  It's the main wiring of the application.